src/constants/constants.ts

```diff
@@ -38,6 +27,18 @@ export const ACTIVATION_EXIT_DELAY = 2 ** 2; // epochs || 25.6 minutes
 export const EPOCHS_PER_ETH1_VOTING_PERIOD = 2 ** 4; // epochs || ~1.7 hours
 export const MIN_VALIDATOR_WITHDRAWAL_DELAY = 2 ** 8; // epochs || ~27 hours
 
+// Initial values
+export const GENESIS_FORK_VERSION	= new BN(0);
```

Comment:
does this actually need to be BN?

Reply:
Good question. I wasn't sure about fork versions, so I just left them as BN.
If we can confirm that we're starting at 0 and incrementing from there, then we could move to `number`.

Reply:
Just asked, it's always 4 bytes so we should be good

Reply:
I.e number should work

Reply:
And they won't necessarily be incremental, rather 4 bytes and we'll need to do some bitwise shifting, the rational was due to potential forks in the future... Although we can always patch this down the road.

It makes more sense as number imo but ultimately doesn't matter

Reply:
Ok! hooray! will update